src/routes/components/[slug]/+page.svelte

```diff
@@ -19,9 +19,9 @@
 	/** @type {import('./$types').PageData} */
 	export let data;
 	let {slug, content, active} = data;
```

Comment:
Is this line still needed?

Reply:
specifically `let {slug, content, active} = data;`

Reply:
Thanks, it works without the line